### ISSUE TITLE
MultipleTimeSeriesCV

### ISSUE THREAD

**Issue by DominicHJ45** [stefan-jansen/machine-learning-for-trading#191]
This isn't a bug but in perusing the code you use  cv=MultipleTimeSeriesCV(...),  imported from utils, but I cannot seem to find  where the code is for this specific definition for cross validation.

**Comment by edesz** [comment]
Hi @DominicHJ45, if you were looking for where the splitter is actually used to perform cross-validation, check out
- chapter 7 notebook 5 ([predicting returns with linear models](https://github.com/stefan-jansen/machine-learning-for-trading/blob/master/07_linear_models/05_predicting_stock_returns_with_linear_regression.ipynb))
  - the splitter used here is slightly modified from the version defined in `utils.py`
- chapter 12 notebook 8 ([predicting returns with gradient boosting libraries](https://github.com/stefan-jansen/machine-learning-for-trading/blob/a84f5e1856b1fdfee3117d0084d0529270baecb4/12_gradient_boosting_machines/08_making_out_of_sample_predictions.ipynb))
  - LGBM using the [training API](https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.train.html#lightgbm.train), since the regressor is defined using `lgb.train(...)`
  - `CatBoostRegressor`, which follows the `sklearn` API ([1](https://catboost.ai/docs/concepts/python-reference_catboostregressor.html#python-reference_catboostregressor), [2](https://catboost.ai/docs/search/?query=provides+compatibility+with+the+scikit-learn+tools))

If you're referring to the location of the source code for the cross-validation splitter, it is defined in the repo's root directory in `utils.py` ([link](https://github.com/stefan-jansen/machine-learning-for-trading/blob/a84f5e1856b1fdfee3117d0084d0529270baecb4/utils.py#L18-L61)), although I think you already saw that version.

**Comment by DominicHJ45** [comment]
Legend, thank you!
